tests/store/tracking/test_file_store_logged_model.py

```diff
@@ -0,0 +1,612 @@
+import time
+from unittest import mock
+
+import pytest
+
+from mlflow.entities.logged_model_parameter import LoggedModelParameter
+from mlflow.entities.logged_model_status import LoggedModelStatus
+from mlflow.entities.logged_model_tag import LoggedModelTag
+from mlflow.exceptions import MlflowException
+from mlflow.store.tracking.file_store import SEARCH_LOGGED_MODEL_MAX_RESULTS_DEFAULT, FileStore
+
```

Comment:
Very comprehensive! 
Can we add (or modify one of these tests) to use `set_experiment` in the place of `create_experiment` to ensure we have FS coverage of the default experiment creation via that API if an experiment doesn't exist?